pangeo-notebook/environment.yml

```diff
@@ -59,7 +58,6 @@ dependencies:
  - numpy
  - pandas
  - panel
- - pangeo-forge-recipes
```

Comment:
if removing these you can either run conda-lock locally to add a new lockfile, or just update all images with the `/condalock` in a comment in this PR.